Update material_request.py

chaoyee opened the pull request:
Add "order by mr_item.item_code ASC" to 'get_material_requests_based_on_supplier(supplier)' in order to get material request item sorted by ascending order.

chaoyee committed:
Update material_request.py

Add "order by mr_item.item_code ASC" to 'get_material_requests_based_on_supplier(supplier)' in order to get material request item sorted by ascending order.

rmehta commented:
@chaoyee thanks for the PR. There are syntax issues in your code. Check if it works!

chaoyee commented:
Oops! Has put the code at wrong place.  Sorry!

chaoyee committed:
Update material_request.py

Add "order by mr_item.item_code ASC" to 'get_material_requests_based_on_supplier(supplier)' in order to get material request item sorted by ascending order.
